riscv/processor.cc

```diff
@@ -351,6 +357,15 @@ void processor_t::parse_isa_string(const char* str)
     }
     p = end;
   }
+  if (extension_table[EXT_ZBPBO]
+   || extension_table[EXT_ZPN]
+   || extension_table[EXT_ZPSFOPERAND]) {
```

Comment:
We prefer the following style for multi-line conditionals; would you mind fixing up these statements accordingly?  https://github.com/riscv-software-src/riscv-isa-sim/blob/ba9f5da8c5f269adf95e34b3c8f434d2084608da/riscv/csrs.cc#L988-L991

Reply:
@aswaterman, is this suitable?

Reply:
Yeah, thanks.